views/js/runner/helpers/map.js

```diff
@@ -204,6 +251,25 @@ define([
         },
 
         /**
+         * Applies a callback on each item of the provided map
+         * @param {Object} map - The assessment test map
+         * @param {Function} callback(item, section, part, map) - A callback to apply on each item
+         * @returns {Object}
+         */
+        each: function each(map, callback) {
+            if (_.isFunction(callback)) {
+                _.each(map && map.parts, function(part) {
```

Comment:
When need to always use the source function over the alias in order to ease the migration to the next version (the aliases are about to change)